### ISSUE TITLE
Missing dependency on `packaging` for 0.4.0 release

### ISSUE THREAD

**Issue by nairb774** [dask/fastparquet#501]
setup.py was not updated to add this dependency. I'm seeing `ModuleNotFoundError: No module named 'packaging'` on a minimal venv. It looks like the import for `packaging` was added in https://github.com/dask/fastparquet/pull/496.

How to reproduce:

```
$ python3 -m venv fastparquet

$ fastparquet/bin/pip install --upgrade pip
Collecting pip
  Using cached https://files.pythonhosted.org/packages/54/2e/df11ea7e23e7e761d484ed3740285a34e38548cf2bad2bed3dd5768ec8b9/pip-20.1-py2.py3-none-any.whl
Installing collected packages: pip
  Found existing installation: pip 19.2.3
    Uninstalling pip-19.2.3:
      Successfully uninstalled pip-19.2.3
Successfully installed pip-20.1

$ fastparquet/bin/pip install fastparquet
Collecting fastparquet
  Downloading fastparquet-0.4.0.tar.gz (152 kB)
     |████████████████████████████████| 152 kB 1.9 MB/s
Collecting pandas>=0.19
  Using cached pandas-1.0.3-cp37-cp37m-macosx_10_9_x86_64.whl (10.0 MB)
Collecting numba>=0.28
  Downloading numba-0.49.1-cp37-cp37m-macosx_10_14_x86_64.whl (2.1 MB)
     |████████████████████████████████| 2.1 MB 7.3 MB/s
Collecting numpy>=1.11
  Using cached numpy-1.18.4-cp37-cp37m-macosx_10_9_x86_64.whl (15.1 MB)
Collecting thrift>=0.11.0
  Using cached thrift-0.13.0.tar.gz (59 kB)
Collecting six
  Using cached six-1.14.0-py2.py3-none-any.whl (10 kB)
Collecting pytz>=2017.2
  Using cached pytz-2020.1-py2.py3-none-any.whl (510 kB)
Collecting python-dateutil>=2.6.1
  Using cached python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
Collecting llvmlite<=0.33.0.dev0,>=0.31.0.dev0
  Downloading llvmlite-0.32.1-cp37-cp37m-macosx_10_9_x86_64.whl (15.9 MB)
     |████████████████████████████████| 15.9 MB 777 kB/s
Requirement already satisfied: setuptools in ./fastparquet/lib/python3.7/site-packages (from numba>=0.28->fastparquet) (41.2.0)
Could not build wheels for fastparquet, since package 'wheel' is not installed.
Could not build wheels for thrift, since package 'wheel' is not installed.
Could not build wheels for setuptools, since package 'wheel' is not installed.
Installing collected packages: numpy, pytz, six, python-dateutil, pandas, llvmlite, numba, thrift, fastparquet
    Running setup.py install for thrift ... done
    Running setup.py install for fastparquet ... done
Successfully installed fastparquet-0.4.0 llvmlite-0.32.1 numba-0.49.1 numpy-1.18.4 pandas-1.0.3 python-dateutil-2.8.1 pytz-2020.1 six-1.14.0 thrift-0.13.0

$ fastparquet/bin/python -c 'import fastparquet'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "fastparquet/lib/python3.7/site-packages/fastparquet/__init__.py", line 5, in <module>
    from .core import read_thrift
  File "fastparquet/lib/python3.7/site-packages/fastparquet/core.py", line 9, in <module>
    from . import encoding
  File "fastparquet/lib/python3.7/site-packages/fastparquet/encoding.py", line 12, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
```

**Comment by martindurant** [comment]
Sorry about that - it was indeed added to the conda recipe. Would you like to submit a PR with the change?

**Comment by martindurant** [comment]
released as 0.4.1
